### DISCUSSION TITLE
How to style the overlay bar?

### DISCUSSION THREAD

**Discussion by aral** [elementary/granite#537] [Q&A]
I’d like to use CSS to style the overlay bar (to adapt it to light/dark schemes as it currently does not).

Any tips?

(I’ve tried using the .overlay-bar class but it has not effect. The only thing I can change is the background of the overlay (not the label) when I use a * {} rule.)

**Comment by danirabbit** [comment]
You can see the styling code from the stylesheet here: https://github.com/elementary/stylesheet/blob/master/src/widgets/_tooltips.scss

Without seeing your code, it's hard to know what's not working
